.github/workflows/phrase-export-checks.yml

```diff
@@ -0,0 +1,84 @@
+name: Check Phrase translations
+on:
+  pull_request:
+    branches:
+      - master
+      - main
+  workflow_dispatch:
+
+jobs:
+  en-keys-to-remove:
+    if: github.head_ref == 'phrase-translations'
+    runs-on: ubuntu-latest
+    name: Check for outdated English keys in Phrase
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Ruby and run bundle install
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Run i18n-tasks
+        run: bundle exec i18n-tasks missing -t diff -f keys en | bundle exec i18n-tasks tree-mv -f keys "en.{*}" "\1" > en-keys-to-remove.txt
+
+      - name: Check that there are zero keys in other locales that don't exist in English
```

Comment:
I had some trouble parsing this. At first I thought this meant "zero keys" as in "keys for count: 0". Maybe:
- "Check that there are no keys in..."
- "Check that all keys exported from Phrase exist in English"